rosetta-source/src/main/resources/cdm-sample-files/functions/fpml-5-10/processes/msg-ex52-execution-advice-trade-partial-novation-C02-00-func-output.json

```diff
@@ -706,26 +706,6 @@
               "globalKey" : "ea328105"
             }
           } ]
-        }, {
-          "meta" : {
-            "externalKey" : "_broker2",
-            "globalKey" : "c3b3523f"
-          },
-          "name" : {
-            "value" : "BROKER 53"
-          },
-          "partyId" : [ {
-            "identifier" : {
-              "meta" : {
-                "scheme" : "http://www.fpml.org/coding-scheme/external/iso17442"
-              },
-              "value" : "549300L7VWW3G1702646"
-            },
-            "identifierType" : "LEI",
-            "meta" : {
-              "globalKey" : "1454247a"
-            }
-          } ]
```

Comment:
This party was present twice in the function output due to a bug having to do with the builder pattern of the generated Java code. This bug has been resolved, so this duplicate was rightfully removed from the expectation.

The bug is actually a symptom of a design issue in the generated build pattern. For details on the root issue, see https://github.com/REGnosys/rosetta-dsl/issues/631.